src/navigator/main-stack.tsx

```diff
@@ -94,6 +96,8 @@ export const MainStackScreen = () => {
   const initLongRefreshLoading = () => {
     dispatch(loadExchangeRates.submit());
     dispatch(loadNotificationsAction.submit());
+    dispatch(loadSwapTokensAction.submit());
+    dispatch(loadSwapDexesAction.submit());
```

Comment:
this should happen only on the swap page.  
no need to update those data if user do not do the swap